docs/walkthrough.md

```diff
@@ -0,0 +1,37 @@
+# Walkthrough
+
+This guide will walk you through creating your first template and workspace. If you haven't already installed `coder`, do that first [here](./install.md).
+
+## Creating your first template and workspace
+
+In a new terminal window, run the following to copy a sample template:
+
+```bash
+coder templates init
+```
+
+Follow the CLI instructions to modify and create the template specific for your
+usage (e.g., a template to **Develop in Linux on Google Cloud**).
+
+Create a workspace using your template:
+
+```bash
+coder create --template="yourTemplate" <workspaceName>
+```
+
+Connect to your workspace via SSH:
+
+```bash
+coder ssh <workspaceName>
+```
+
+## Modifying templates
```

Comment:
Didn't know where to put this so left it in walkthrough 🤷🏼‍♂️

Reply:
Can we call it "quickstart"? I've seen that term used across different project docs pages

Reply:
Yes! I'll change that now.